README.md

```diff
@@ -1,3 +1,36 @@
+# Recognition of human positions
```

Comment:
It sounds too academic. I think we can describe it is much simpler. Just attach some screenshots of application, and describe it in a few sentences. Also I suggest adding a link to the `posenet` npm package that is used in application.